virttest/utils_test/__init__.py

```diff
@@ -1969,29 +1969,65 @@ def _git_download(self, url, destination):
         destination = os.path.join(destination, self.base_name)
         git.get_repo(url, destination_dir=destination)
 
-    def _tarball_download(self, url, destination):
+    def _tar_extract(self, tar_file, destination):
```

Comment:
@balamuruhans it would be good if we reuse, this from avocado `avocado/utils/archive.py`, looks like this supports most of archive types.

Reply:
@sathnaga sure, I will make the change. Thank you for review.